packages/relay/src/RegistrationManager.ts

```diff
@@ -559,7 +559,8 @@ Hub authorized | ${boolString(this.isHubAuthorized)}
 Stake locked   | ${boolString(this.isStakeLocked)}
 Manager        | ${this.managerAddress}
 Worker         | ${this.workerAddress}
-Owner          | ${this.ownerAddress ?? chalk.red('unknown')}
+Stake Owner    | ${this.ownerAddress ?? chalk.red('not set')}
```

Comment:
empty stakeOwner is "yellow" (its a transient state not error during setup, and once a balance is put in, it will be fixed automatically)